src/views/blocks/tickets/submit-button.php

```diff
@@ -53,5 +53,5 @@
 	<?php endif; ?>
 	<?php tribe_disabled( true ); ?>
 >
-	<?php echo esc_html_x( 'Get Tickets', 'Add tickets to cart.', 'event-tickets' ); ?>
+	<?php echo esc_html( sprintf( _x( 'Get %s', 'Add tickets to cart.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets') ) ); ?>
```

Comment:
Needs 'translators' placeholder doc

Reply:
This view also needs `@version TBD` and `@since TBD Use tribe_get_ticket_label_plural() for "tickets" string`. Also there's a whitespace formatting issue here.

```suggestion
	<?php echo esc_html( sprintf( _x( 'Get %s', 'Add tickets to cart.', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) ) ); ?>
```